chapter_1/exercise_1_12/copy_io_nl.c

```diff
@@ -2,22 +2,21 @@
 
 int main()
 {
-  char c;
-  char last_char = -1;
+  int c, prev;
+  prev = EOF;
```

Comment:
@caldotdev You can do initialization during declaration, so you don't need this additional line here. Also, I would appreciate improving the naming of the `prev` variable to `previous_char`. It might seem like a long name, but I would advocate that it improves readability, especially for people who try to understand the solution more easily.

Reply:
wouldn't be

```c
int char
int last_char
```

be more intuitive and consistent here?